src/webpack.ts

```diff
@@ -20,7 +20,8 @@ export function getWebpackConfig (config: webpack.Configuration, extras: Webpack
   ]
   const storybookValidPlugins = [
     'VirtualModulesPlugin',
-    'HtmlWebpackPlugin'
+    'HtmlWebpackPlugin',
+    'DefinePlugin' // Required because of https://storybook.js.org/docs/vue/configure/environment-variables
```

Comment:
Nuxt also injects via `env` option but it is different because not automatically injecting everything. So something may work with storybook but not nuxt project. Is that expected?